src/main.rs

```diff
@@ -1405,12 +1399,8 @@ fn parse_and_output_man_pages(
             deroff_only,
             write_to_stdout,
         ) {
-            Ok(successful) => {
-                if successful {
-                    successful_count += 1
-                }
-                // Theres no else here, i assume they just pass over it lol
-            }
+            Ok(true) => successful_count += 1,
+            Ok(false) => (),
```

Comment:
There's a clippy lint to change this to `{}` later, so we can just use `{}` rather than `(),`. I forgot which one is it already.